treeherder/intermittents_commenter/management/commands/run_intermittents_commenter.py

```diff
@@ -1,16 +1,27 @@
+import calendar
+from datetime import date
+
 from django.core.management.base import BaseCommand
 
 from treeherder.intermittents_commenter.commenter import Commenter
 
 
 class Command(BaseCommand):
```

Comment:
Just remembered - could you add `run_intermittents_commenter` to the list of one-off commands that New Relic annotates, here?
https://github.com/mozilla/treeherder/blob/964d81f8ac0a79fc5fd2ac28d4462a969d405ccd/newrelic.ini#L28-L31